inference-engine/src/gna_plugin/runtime/pwl.cpp

```diff
@@ -499,82 +500,95 @@ std::vector<pwl_t> pwl_search(const DnnActivation& activation_type,
 
 void PwlDesignOpt16(const DnnActivation activation_type,
                     std::vector<gna_pwl_segment_t> &ptr_segment,
-                    const float scale_in,
-                    const float scale_out) {
+                    GNAPluginNS::QuantizedLayerParams const* quantParams) {
```

Comment:
pwl design was less dependen on GNA structures, so i dont think that adding one more reference make sense at all.

Reply:
let me try to remove it